### PR TITLE
Use CodeCov

### LINKED ISSUE #93: proposal: Use CodeCov
CodeCov is SaaS to visualize code coverage, and useful to improve code quality.

Note that the below samples refer my personal repositories.

* About CodeCov: https://docs.codecov.io/docs/about-code-coverage
* Sample status badges: [codecov](https://codecov.io/gh/suzuki-shunsuke/go-set) [codecov](https://codecov.io/gh/suzuki-shunsuke/go-graylog)
* Sample pages
  * https://codecov.io/gh/suzuki-shunsuke/go-graylog
  * https://codecov.io/gh/suzuki-shunsuke/go-set
  * https://codecov.io/gh/suzuki-shunsuke/go-graylog/src/master/client/stream_rule.go
* Sample [Pull Request Comments](https://docs.codecov.io/docs/pull-request-comments)
  * https://github.com/suzuki-shunsuke/go-graylog/pull/8#issuecomment-401587635

### PR DESCRIPTION
**Pull request by suzuki-shunsuke**
Close #93

https://codecov.io/gh/line/line-bot-sdk-go
https://github.com/codecov/example-go#caveat-multiple-files

### CODE REVIEW

**Comment by suzuki-shunsuke** [comment]
https://codecov.io/gh/line/line-bot-sdk-go/tree/14fa35219c759a9a66c647cb0aea54825508e98e/linebot

File: script/codecov.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# https://github.com/codecov/example-go#caveat-multiple-files
+
+cd `dirname $0`/.. || exit 1
+echo "" > coverage.txt || exit 1
+for d in $(go list ./... | grep -v vendor); do
+  go test -race -coverprofile=profile.out -covermode=atomic $d || exit 1
```
Comment (sugyan) [inline script/codecov.sh]: Do not use `-v` option?

Comment (suzuki-shunsuke) [reply script/codecov.sh]: Thank you for your review.
I have added `-v` option.
https://github.com/line/line-bot-sdk-go/pull/142/commits/258ad236b899b8b07131e5ac66b5be789aeb3040

Comment (sugyan) [reply script/codecov.sh]: Thanks! LGTM 🙆‍♂️

**Pull request merged**